### DISCUSSION TITLE
`tinacms build` doesn't load .env.production

### DISCUSSION THREAD

**Discussion by psirenny** [tinacms/tinacms#3355] [Ideas]
## Description

`tinacms build` doesn't load environment variables from `.env.production`.

### Expected result

Expected `tinacms build` to follow the same environment variable loading rules as `next build`.

### What should happen?

`tinacms build` should behave consistently with `next build` and load environment variables in the [following order](https://nextjs.org/docs/basic-features/environment-variables#environment-variable-load-order):
| Location  | Works |
| ------------- | ------------- |
| `process.env`  | ✅  |
| `.env.production.local`  | ❌ |
| `.env.local`  | ❌ |
| `.env.production`  | ❌ |
| `.env`  | ✅

**Comment by jamespohalloran** [comment]
Just replaced the bug label with enahancement. I agree it would be nice to match Next's env behaviour here, but for context, we use [dotenv](https://www.npmjs.com/package/dotenv) under the hood. In their FAQ, it says:

> Should I have multiple .env files?
> No. We strongly recommend against having a "main" .env file and an "environment" .env file like .env.test. Your config should vary between deploys, and you should not be sharing values between environments.
